src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWriter.cs

```diff
@@ -153,7 +153,7 @@ public void WriteStartAttribute(string localName)
 
         // Writes out a processing instruction with a space between the name and text as follows: <?name text?>
 
-        public abstract void WriteProcessingInstruction(string name, string? text);
+        public abstract void WriteProcessingInstruction(string name, string text);
```

Comment:
I believe I changed it after I found couple of implementations clearly not expecting it to be null, since it's documented I'll probably just change it to treat empty and null the same

Reply:
I wrote a comment explaining what doesn't expect null (this is likely can be replaced with string.Empty but we need to be careful to not change semantics so I'll leave this out of this PR)